README.md

```diff
@@ -675,6 +675,22 @@ request.get({
 });
 ```
 
+The `ca` value can be an array of certificates, in the event you have a private or internal corporate public-key infrastructure hierarchy. For example, if you want to connect to https://api.some-server.com which presents a key chain consisting of:
+1. its own public key, which is signed by:
+2. an intermediate **some-server Corp Issuing Server 01**, that is in turn signed by: 
+3. a root CA **some-server Corp Root CA**;
+
+you can configure your request as follows:
+
+```js
+request.get({
+    url: 'https://api.some-server.com/',
```

Comment:
Some of this is confusingly worded, I'd recommend simplifying to:
```
1. its own public key, which is signed by:
2. an intermediate *Corp Issuing Server*, that is in turn signed by: 
3. a  *Corp Root CA*.
```

Note the switch away from bold, which isn't needed to draw attention here. Switch to either italics or double quotes.